### PR TITLE
feat: create switch component

### PR DESCRIPTION
**Pull request by mjmonline**

### CODE REVIEW

File: src/Molecules/Switch/Switch.tsx [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+      border: 1px solid ${p => p.theme.color.disabledBackground};
+    }
+  }
+`;
+
+const generateSomewhatUniqueId = (str: string) => {
+  const safeString = str.replace(/^[^a-z]+|[^\w:.-]+/gi, '').toLowerCase();
+
+  return `switch-${safeString}`;
+};
```
Comment (iamstarkov) [inline src/Molecules/Switch/Switch.tsx]: maybe use https://github.com/ai/nanoid instead

File: src/Molecules/Switch/Switch.types.ts [hunk 2]
```diff
@@ -0,0 +1,7 @@
+export type Props = {
+  className?: string;
+  defaultOn?: boolean;
```
Comment (shmidt-i) [inline src/Molecules/Switch/Switch.types.ts]: What about controllable vs uncontrollable? 
Currently this component is 100% uncontrollable, which may hit us in some time, we probably want both behaviours to exist

File: src/Molecules/Switch/Switch.tsx (hunk 1 above)
Comment (mjmonline) [reply src/Molecules/Switch/Switch.tsx]: I checked up the use of <label> outside <form> and it seems ok. So now I wrap the switch in a <label> which allows me to not need any id

File: src/Molecules/Switch/Switch.types.ts (hunk 2 above)
Comment (mjmonline) [reply src/Molecules/Switch/Switch.types.ts]: @shmidt-i added support for this

File: src/Molecules/Switch/Switch.tsx (hunk 1 above)
Comment (iamstarkov) [reply src/Molecules/Switch/Switch.tsx]: that's good news

File: src/Molecules/Switch/Switch.types.ts [hunk 3]
```diff
@@ -0,0 +1,12 @@
+export type Props = {
+  className?: string;
+  checkedInitially?: boolean;
+  /**
+   * Using this prop will enable controlled behaviour
+   * https://reactjs.org/docs/forms.html#controlled-components
+   */
+  checked?: boolean;
+  disabled?: boolean;
+  labelText: string;
```
[the remark this answers is not in the document]
Comment (mjmonline) [reply src/Molecules/Switch/Switch.types.ts]: Done

**Pull request merged**